test/manipulation.test.js

```diff
@@ -934,6 +956,23 @@ describe('manipulation', function() {
         });
       });
 
+      it('should have updated password hashed with replaceAttributes',
+      function(done) {
+        StubUser.create({ password: 'foo' }, function(err, created) {
+          if (err) return done(err);
+          created.replaceAttributes({ password: 'test' }, { validate: false },
```

Comment:
Please see the fix: https://github.com/strongloop/loopback-datasource-juggler/pull/896

I belive https://github.com/strongloop/loopback-datasource-juggler/pull/896 needs to be landed before this patch#893